docker-compose.yml

```diff
@@ -5,9 +5,20 @@ services:
     build:
       context: .
       target: ${DOCKER_TARGET:-development}
-      dockerfile: Dockerfile
+      dockerfile: docker/ahti-ui
     volumes:
       - .:/app
       - /app/node_modules
     ports:
       - 3000:3000
+    depends_on:
+      - faker
+  faker:
```

Comment:
Just dry coding here, but you might be able to do this without adding a separate Dockerfile.

```
faker:
    image: apisguru/graphql-faker
    command: ["/app/graphql-faker/schema.sdl"]
    container_name: ahti-graphql-faker-api
    volumes:
      - .:/app
    ports:
      - 9002:9002

Reply:
Indeed it can be done just by supplying a command to the image. I was thinking whether we want to pull an external image or just supply our own dockerfile, as it is very simple. Though I guess this is the simpler approach.

Also changed the volume.